src/focus.js

```diff
@@ -0,0 +1,37 @@
+﻿import {customAttribute} from 'aurelia-templating';
+import {inject} from 'aurelia-dependency-injection';
+
+var afterAttachedSetFocusDelay = 50;
+
+@customAttribute('focus')
+@inject(Element)
+export class Focus {
+    constructor(element) {
+        this.element = element;
+
+        this.focusListener = (e => this.value = true).bind(this);
+        this.blurListener = (e => this.value = false).bind(this);
```

Comment:
I tried making the event handlers methods: ```onFocus(e) { this.value = true; }```.

If I attach them this way: ```this.element.addEventListener('focus', this.onFocus);``` then when the event is fired, the ```this``` keyword refers to the HTML element itself, not the ```Focus``` instance, so it doesn't work.

If I attach them this way: ```this.element.addEventListener('focus', e => this.onFocus(e));``` then when doing the detach ```this.element.removeEventListener('focus', e => this.onFocus(e));``` the handler is not correctly removed. I think this is because of the es5to6 transpiler, which creates a ```_this2``` variable in ```attached()``` and a ```_this3``` variable in ```detached()```, but I'm not 100% sure.

So unless there's another solution which I'm not seeing, I think that how I did it is the simplest working solution :)